walk-in-interview/src/main/webapp/account/create-account/business/account-info/script.js

```diff
@@ -0,0 +1,136 @@
+const CURRENT_LOCALE = 'en';
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../../../../strings.en.js';
+import {API} from '../../../../apis.js';
+import {Auth} from '../../../firebase-auth.js';
+import {USER_TYPE_COOKIE_PARAM, USER_TYPE_BUSINESS,
+  setCookie, setErrorMessage} from '../../../../common-functions.js';
+
+const HOMEPAGE_PATH = '../../../../index.html';
+const STRINGS = AppStrings['create-business-account'];
+const ACCOUNT_STRINGS = AppStrings['create-account'];
+const BAD_REQUEST_STATUS_CODE = 400;
+
+window.onload = () => {
+  Auth.subscribeToUserAuthenticationChanges(
+    onLogIn, onLogOut, onLogInFailure, onLogOutFailure);
+  renderPageElements();
+};
+
+/**
+ * What to do after the user signed in and the session cookie is created.
+ */
+function onLogIn() {
+  // TODO(issue/100): set the cookie at the server side instead
+  setCookie(USER_TYPE_COOKIE_PARAM, USER_TYPE_BUSINESS);
+}
+
+function onLogOut() {
+  // TODO(issue/101): Display button according to log in status;
+}
+
+function onLogInFailure() {
+  // TODO(issue/101): Display button according to log in status;
+}
+
+
+function onLogOutFailure() {
+  // TODO(issue/101): Display button according to log in status;
+}
+
+
+/** Adds all the text to the fields on this page. */
+function renderPageElements() {
+  const submitButton = document.getElementById('submit');
+  submitButton.setAttribute('value', ACCOUNT_STRINGS['submit']);
+  submitButton.setAttribute('type', 'submit');
+
+  const nameLabel = document.getElementById('name-label');
+  nameLabel.innerText = STRINGS['name'];
+
+  const name = document.getElementById('name');
+  name.setAttribute('type',  'text');
+}
+
+/**
+ * Gets account detail from user input.
+ * 
+ * @return {Object} Business account object containing the user inputs.
+ */
+function getBusinessDetailsFromUserInput() {
+  const businessName = document.getElementById('name').value.trim();
+
+  const businessDetails = {
+    userType: USER_TYPE_BUSINESS,
+    name: businessName,
+    // empty job list is created at the server
+  };
+
+  return businessDetails;
+}
+
+/**
+* Validates the user input.
+* Shows error message on the webpage if there is field with invalid input.
+*
+* @return {boolean} depending on whether the input is valid or not.
+*/
+function validateRequiredUserInput() {
+  const name = document.getElementById('name').value.trim();
+
+  if (name === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+        /* msg= */ STRINGS['name']);
+    return false;
+  }
+
+  return true;
+}
+
+const submitButton = document.getElementById('submit');
+submitButton.addEventListener('click', (_) => {
+  if (!validateRequiredUserInput()) {
+    return;
+  }
+
+  const accountDetails = getBusinessDetailsFromUserInput();
+
+  fetch(API['create-business-account'], {
```

Comment:
To avoid confusion, maybe add another same endpoint with different naming
" fetch(API['update-business-account']"
Or add a comment here saying // Update account info.
Or else this could be mistaken as account is created in this page.